packages/api/src/routes/internal/medical/patient-import.ts

```
@@ -213,15 +216,28 @@ router.post(
     const updateParams = updateJobSchema.parse(req.body);
     capture.setExtra({ cxId, jobId });
 
-    const patientImport = await updatePatientImportTracking({
+    if (updateParams.status === "waiting") {
+      throw new BadRequestError("Waiting status is not allowed on this endpoint");
+    }
+    const { status: jobStatus, reason } = updateParams;
+    if (reason && jobStatus !== "failed") {
```

The reason simply won't be used if the job is set to failed -- do we want it to fail silently or throw? NABD either way